doc/content/integrations/mqtt-clients/eclipse-mosquitto/_index.md

```diff
@@ -33,6 +33,8 @@ For example, to subscribe to all topics in the application `app1`:
 $ mosquitto_sub -h thethings.example.com -t "#" -u app1 -P "NNSXS.VEEBURF3KR77ZR.." -d
 ```
 
+{{< note >}} For details about using this command in a multi-tenant environments like {{% tts %}} Cloud or Community Edition, see the [Multi-tenancy section of the MQTT Server guide](http://localhost:1313/integrations/mqtt#multi-tenancy). {{</ note >}}
```

Comment:
The Community Edition isn't multi-tenant really.

Basically, in every edition but The Things Stack Open Source, one needs to pass a tenant ID. But that doesn't mean the deployment is multi-tenant. TTN is a single tenant deployment.

Reply:
Also use `relref` or `ref`